doc/sphinxman/source/build_planning.rst

```diff
@@ -0,0 +1,1152 @@
+compiling. Note that these flags *will not* propagate to any add-ons that
+are detected pre-built rather than built.
+
+* Build without optimization
+
+  .. code-block:: bash
+
+    cmake -DCMAKE_BUILD_TYPE=debug
+
+                                  set the CMake build type [default: release]
+
+* Relevant CMake Options:
+
+  .. code-block:: bash
+
+    CMAKE_BUILD_TYPE=[debug|release]  # Build type (Release or Debug)" [default: release]
+
+
+
+.. .. _`faq:setupobjdir`:
+.. 
+.. How to compile elsewhere than ``{top-level-psi4-dir}/objdir``
+.. -------------------------------------------------------------
+.. 
+.. [How to choose the compilation directory, ``$objdir``](2_Planning#chooseobjdir)
+.. 
+.. * Build in Specific Directory
+.. 
+..   .. code-block:: bash
+.. 
+..    cd $top-level-psi4-dir
+..    cmake -H. -Bobj-gcc
+..    cd obj-gcc
+
+
+.. _`faq:erroreriam`:
+
+How to fix error "``RuntimeError: value for ERI``"
+--------------------------------------------------
+
+You will need to rebuild libint. Reissue ``cmake`` or edit
+``CMakeCache.txt`` with larger ``MAX_AM_ERI`` and rebuilt.
+
+* :ref:`faq:setupmaxameri`
+
+.. _`faq:chooseobjdir`:
+
+How to choose the compilation directory, ``{objdir}``
+-----------------------------------------------------
+
+* there is no default
+* common choices are ``objdir`` or ``build`` under :samp:`{top-level-psi4-dir}`
+
+  * ``cd {top-level-psi4-dir} && cmake -H. -Bobjdir``
+  * ``cd {top-level-psi4-dir} && cmake -H. -Bbuild``
+
+* in-source builds (``*.cc`` and ``*.o`` in same directory) are disallowed
+* builds *outside* :samp:`{top-level-psi4-dir}` are permitted
+
+
+.. _`faq:doconfigure`:
+
+How to save configuration settings for a future compilation
+-----------------------------------------------------------
+
+Create a file like ``do-configure`` with the ``cmake`` command and options
+*on one line*. ::
+
+ >>> cd {top-level-psi4-dir}
+ >>> cat do-configure
+     cmake -H. -Bobjdir5 \
+         -DCMAKE_INSTALL_PATH="/Users/me/psi4" \
+         -DCMAKE_PREFIX_PATH="/Users/me/externals/install-libint" \
+         -DMAX_AM_ERI=6 \
+         -DENABLE_gdma=ON \
+         -DBUILD_SHARED_LIBS=ON
+ >>> chmod u+x do-configure
+ >>> ./do-configure
+
+
+.. _`faq:dirlayoutinstall`:
+
+What is the directory layout of the installed or staged Psi4
+------------------------------------------------------------
+
+After compilation (:samp:`cd {objdir} && make`), a directory structure like the
+below will exist at :samp:`{objdir}/stage/{prefix}`. This may be tested and used
+just like a full installation.
+
+After installation (:samp:`cd {objdir} && make && make install`), a directory
+structure like the below will exist at :samp:`/{prefix}`. This is a full
+installation.
+
+.. code-block:: bash
+
+ /
+ bin/                                                   (executables fro psi4 + any external proj)
+ bin/psi4                                                      (py script masquerading as c++ exe)
```

Comment:
I'd like it to be a little advanced. Switched to `psi4 executable, actually just a py script`